internal/graph/check.go

```diff
@@ -722,12 +722,15 @@ func TraitsForArrowRelation(ctx context.Context, reader datastore.Reader, namesp
 }
 
 func queryOptionsForArrowRelation(ctx context.Context, ds datastore.Reader, namespaceName string, relationName string) ([]options.QueryOptionsOption, error) {
+	opts := []options.QueryOptionsOption{
+		options.WithQueryShape(queryshape.AllSubjectsForResources),
+	}
+
 	traits, err := TraitsForArrowRelation(ctx, ds, namespaceName, relationName)
 	if err != nil {
-		return nil, err
+		return opts, err
```

Comment:
Why not nil here?

Reply:
Just figured we'd always returned the query shape, even if there was an error

Reply:
that's not idiomatic